india_compliance/gst_india/overrides/subcontracting_transaction.py

```diff
@@ -69,11 +69,18 @@ def after_mapping_subcontracting_order(doc, method, source_doc):
 
 
 def after_mapping_stock_entry(doc, method, source_doc):
-    if source_doc.doctype == "Subcontracting Order":
+    if not source_doc.doctype == "Subcontracting Order":
+        doc.taxes_and_charges = ""
+        doc.taxes = []
+
+    if not doc.purpose == "Material Transfer" or not doc.is_return:
```

Comment:
- This will not work currently as the `purpose` and all related fields are populated after the doc is mapped.

- So, either we have to populate those fields in `post process` or do it some other way.

Reply:
Reference PR in ERPNext: https://github.com/frappe/erpnext/pull/43061